beacon_node/http_api/src/lib.rs

```diff
@@ -3246,6 +3247,46 @@ pub fn serve<T: BeaconChainTypes>(
             },
         );
 
+    // POST lighthouse/liveness/{epoch}
```

Comment:
The path should be `/eth/v1/validator/liveness/{epoch}`, matching the spec: https://ethereum.github.io/beacon-APIs/#/Validator/getLiveness